invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsers/AccessUsers.js

```diff
@@ -57,10 +59,33 @@ export class AccessUsers extends Component {
       console.error(error);
     }
   };
+
+  fetchOwner = async () => {
+    const { record } = this.props;
+    this.setState({ loading: true });
+
+    const ownerId = record.parent?.access?.owned_by?.user;
```

Comment:
I think you might be able to `expand` the `access.owned_by.user` in the [existing request in fetchData](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessUsers/AccessUsers.js#L43)

Reply:
In the `fetchData` we are sending the request to get all the grants. Record owner doesn't have any grants, so that request doesn't return anything about record owner. That's why I had to fetch it separately. Please let me know if i'm missing something here..

Reply:
Hey, maybe I didn't explain myself. I didn't mean to expand the owner on the grant request i.e `fetchData` but if I remember correctly the owned_by can be expanded when you get the record. We can have a quick call if there is still some confusion.

Reply:
Added expanded field 🚀